modules/core/src/main/scala/ch/epfl/bluebrain/nexus/rdf/Graph.scala

```diff
@@ -1,29 +1,42 @@
 package ch.epfl.bluebrain.nexus.rdf
 
+import cats.Eq
 import cats.implicits._
 import ch.epfl.bluebrain.nexus.rdf.Graph._
 import ch.epfl.bluebrain.nexus.rdf.Iri.Path.Segment
 import ch.epfl.bluebrain.nexus.rdf.Iri.{AbsoluteIri, Url}
 import ch.epfl.bluebrain.nexus.rdf.Node.{BNode, IriNode, IriOrBNode}
 
+/**
+  * An RDF Graph anchored to a specific node. All vertices except the anchor are existential. Operations like append
```

Comment:
What does `existential` mean in this context?

Reply:
The vertices exist through the edge definitions only, cannot be asserted independently.